### ISSUE TITLE
vHandle shapelet components in srclist-by-beam with ao output

### ISSUE THREAD

**Issue by d3v-null** [MWATelescope/mwa_hyperdrive#1]
example:

using `srclist_pumav3_EoR0aegean_fixedEoR1pietro+ForA_phase1+2` from <https://github.com/JLBLine/srclists> 

```bash
export srclist="path/to/srclist.txt"
hyperdrive srclist-by-beam \
                 -n 100 \
                 -o ao \
                 -m ${obsid}.metafits \
                 "${HOME}/srclists/${srclist}.txt"
```

I get the error

**Comment by cjordan** [comment]
This should be fixed now in the latest DI commit (I keep force pushing it, but I won't do any more for that commit... sorry!). I'll leave this open until it's confirmed.

**Comment by cjordan** [comment]
I just checked this and no warning is issued; I'll keep the issue open until that's done.

**Comment by cjordan** [comment]
Fixed in a6432aa3e53c2a2f54cb77f873caf212b4caca26
